FourthTask/sorting/types/types.go

```diff
@@ -0,0 +1,40 @@
+//	Package contains person type and implements sort.Interface
+package types
+
+import (
+	"time"
+)
+
+var (
+	StartDate time.Time = time.Date(0, time.January, 1, 0, 0, 0, 0, time.UTC) //user will set data relatively Jesus birthday
+)
+
+type Person struct {
+	FirstName string
+	LastName  string
+	Birthday  time.Time
+}
+
+type People []Person
+
+func (p People) Len() int {
+	return len(p)
+}
+
+func (p People) Less(i, j int) bool {
+
+	diffDays := p[j].Birthday.Sub(p[i].Birthday) / 24
```

Comment:
I bet this should work, but you can use built-in magic https://golang.org/pkg/time/#Time.Before

Reply:
in this task i can't use this func. because we have 3 cases here.

Reply:
1st check if equals, 2nd if Before, for example. Anyway, it's up to you which way to prefer.